ods_ci/tests/Resources/Files/cluster-monitoring-config.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: cluster-monitoring-config
+    namespace: openshift-monitoring
+data:
+    config.yaml: |
+        enableUserWorkload: true
```

Comment:
we already have this file in the repo: `ods_ci/tests/Resources/Files/llm/uwm_cm_enable.yaml` (and related `ods_ci/tests/Resources/Files/llm/uwm_cm_conf.yaml`).

You can move them outside the `llm` folder so we don't have unnecessary duplicates. If you do so, please change accordingly `ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot` (it should be a 2 lines change)

Reply:
Moved the files outside and updated the 422__model_serving_llm.robot